reference/3.0/PSDiagnostics/Disable-WSManTrace.md

```diff
@@ -1,44 +1,60 @@
 ---
 external help file: PSDiagnostics-help.xml
+keywords: powershell,cmdlet
+locale: en-us
 Module Name: PSDiagnostics
-online version:
+ms.date: 11/29/2018
 schema: 2.0.0
 ---
 
 # Disable-WSManTrace
 
 ## SYNOPSIS
-{{Fill in the Synopsis}}
+Stop the WSMan logging session started by Enable-WSManTrace
 
 ## SYNTAX
 
 ```
-Disable-WSManTrace
+Disable-WSManTrace [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-{{Fill in the Description}}
+This cmdlet stops the WSMan logging session started by Enable-WSManTrace.
+
+This cmdlet uses the `Stop-Trace` cmdlet.
+
+You must run this cmdlet from an elevated PowerShell session.
 
 ## EXAMPLES
 
-### Example 1
-```powershell
-PS C:\> {{ Add example code here }}
+### Example 1: Stop a WSMan trace
+```
```

Comment:
Does there need to be a blank line between the Example header and the codefence?  There is in other files in this PR.

Reply:
Fixed